IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -148,6 +148,8 @@ def agg_csv(advanced_im_config, stations, im_calc_dir):
 
 def natural_key(string_):
     """
-    using regex to get a sequence of numbers in a string and turn thme into int for sorting purpose
+    using regex to get a sequence of numbers in a string and turn them into int for sorting purpose
+    example strings: test_model_v20p1p1  test_model2_v20p1p11
+    keys: [test_model_v, 20, p, 1, p , 1],  [test_model2_v, 20, p, 1, p, 11]
```

Comment:
put quotes on here to indicate which are strings and which are ints.

Is the second example incorrect ```test_model2_v```

Reply:
haha, yeah, i noticed it when i was reading it this morning. updated.